graph.go

```diff
@@ -108,6 +108,17 @@ func (g *Graph) Query(q string) (*QueryResult, error) {
 	return QueryResultNew(g, r)
 }
 
+// QueryRO executes a read only query against the graph.
+func (g *Graph) QueryRO(q string) (*QueryResult, error) {
```

Comment:
Looks good, but please rename from `QueryRO` to `RO_QUERY` to be aligned with the command name

Reply:
Ack, will do. To clarify, `RO_QUERY` or `ROQuery`?

Reply:
@swilly22 I've renamed this to ROQuery to be more in line with the current `Query` function. Let me know if you agree/disagree and I can merge if you're okay with this.

Reply:
Can't we go with `RO_Query` ?

Reply:
Sure. PR update incoming in a bit.